pkg/virt-handler/isolation/detector.go

```diff
@@ -119,7 +119,7 @@ func (s *socketBasedIsolationDetector) Allowlist(controller []string) PodIsolati
 
 func (s *socketBasedIsolationDetector) AdjustResources(vm *v1.VirtualMachineInstance) error {
 	// only VFIO attached or with lock guest memory domains require MEMLOCK adjustment
-	if !util.IsVFIOVMI(vm) && !vm.IsRealtimeEnabled() {
+	if !util.IsVFIOVMI(vm) && !vm.IsRealtimeEnabled() && !util.IsSEVVMI(vm) {
```

Comment:
Out of curiosity, does the use of a secure launch with QEMU generate an additional memory overhead?

Reply:
Indeed libvirt documentaiton suggests to add `256MiB` extra memory. Seems that I forgot about it at some point. Thanks, good catch :+1: